eng/Versions.props

```diff
@@ -55,7 +55,7 @@
     <MicrosoftNETSdktvOSManifest90100PackageVersion>18.4.9288</MicrosoftNETSdktvOSManifest90100PackageVersion>
     <MicrosoftNETSdkMacCatalystManifest90100PackageVersion>18.4.9288</MicrosoftNETSdkMacCatalystManifest90100PackageVersion>
     <MicrosoftNETSdkmacOSManifest90100PackageVersion>15.4.9288</MicrosoftNETSdkmacOSManifest90100PackageVersion>
-    <MicrosoftNETSdkMauiManifest90100PackageVersion>9.0.14</MicrosoftNETSdkMauiManifest90100PackageVersion>
+    <MicrosoftNETSdkMauiManifest90100PackageVersion>9.0.51</MicrosoftNETSdkMauiManifest90100PackageVersion>
```

Comment:
Double checking, this should be 51 and not 15 - just seemed like a big jump

Reply:
correct, they increment with every build so often jump multiple at a time. Here was the official codeflow: https://github.com/dotnet/workload-versions/pull/387